ObjectsComparer/ObjectsComparer/BaseComparer.cs

```diff
@@ -22,7 +22,9 @@ public abstract class BaseComparer: IBaseComparer
 
         protected IComparersFactory Factory { get; }
 
-        internal ComparerOverridesCollection OverridesCollection { get;  } = new ComparerOverridesCollection();
+        internal ComparerOverridesCollection OverridesCollection { get; } = new ComparerOverridesCollection();
+
+        internal MemberIgnoreCollection IgnoreCollection { get; } = new MemberIgnoreCollection();
```

Comment:
Why don't add override for the member using DoNotCompareValueComparer.Instance instead of creating additional collection?

Reply:
Yeah that's a lot simpler. As I was doing this, I'm kind of confused about what some of these pieces do. Namely, I don't understand how the Filter lambda on the ValueComparerWithFilter is being used.

I am tempted to use that for this but I'm not sure what it's currently being used for.

Reply:
I'll have a look tomorrow :)